xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -512,6 +516,74 @@ rebuild_master:
         configure_repositories step, set the <emphasis>add_repositories</emphasis> key to anything else than <emphasis>enabled</emphasis>.</para>
         </sect4>
 
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.submit_request">
+          <title>Submit a Request</title>
+
+          <para>The submit request step is the equivalent of the <emphasis>osc submitrequest</emphasis> command.</para>
+
+          <para>The requirements to run a submit request step are:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>There must be a branch created.</para>
+            </listitem>
+
+            <listitem>
+              <para>There has to be some changes in the package branch.</para>
+            </listitem>
+          </itemizedlist>
+
+          <para>After the previous requirements are met, keep in mind that:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>With a pull request open event, push event, or tag_push event, it will:</para>
+              <itemizedlist>
+                <listitem>
+                  <para>create the submit request.</para>
+                </listitem>
+
+                <listitem>
+                  <para>report further build results to the pull request.</para>
+                </listitem>
+              </itemizedlist>
+            </listitem>
+
+
+            <listitem>
+              <para>With a pull_request synchronize event, it will:</para>
```

Comment:
```suggestion
              <para>When receiving a pull_request synchronize event, it will:</para>
```

Reply:
I also think we should translate `pull_request synchronize event` into something easier to understand. Maybe `When more commits are added to the pull request`?